src/main/java/com/google/codeu/utils/TextUtils.java

```diff
@@ -0,0 +1,71 @@
+/*
+package com.google.codeu.servlets;
```

Comment:
Never submit code that is commented out unless it is intended to be a comment. If you need the class to be in the develop branch consider writing a skeleton implementation that can be built upon. You are allow to make the class and have methods that literally return nothing or no-op. You can make a branch off of your current branch 

1. git checkout klin_StyledText
2. git checkout -b klin_StyledText2

klinStyledText2 will have the changes from klin_StyledText and this will allow you to make multiple pull requests even if StyledText2 relies on StyledText1